.cci.jenkinsfile

```diff
@@ -72,7 +72,10 @@ parallel vmcheck: {
         // really, we just need test deps, but meh...
         shwrap("ci/installdeps.sh")
       }
-      stage("Run") {
+      stage("Kola") {
+        fcosKola(cosaDir: "${env.WORKSPACE}", extraArgs: "ext.*", parallel: nhosts)
```

Comment:
Actually, can we also do `skipUpgrade: true` here?

Reply:
We can do as a follow-up too.

Not sure why the blueocean UI is missing the vmcheck stage entirely, but it is running it.